src/main/java/org/openforis/ceo/local/JsonGeoDash.java

```diff
@@ -84,9 +84,10 @@ public synchronized String updateDashBoardById(Request req, Response res) {
 
     
     public synchronized String createDashBoardWidgetById(Request req, Response res) {
-        var dashboardId = req.queryParams("dashID");
-        var widgetJson = req.queryParams("widgetJSON");
-        var callback = req.queryParams("callback");
+        var jsonInputs            = parseJson(req.body()).getAsJsonObject();
+        var dashboardId = jsonInputs.get("dashID").getAsString();
+        var widgetJson = jsonInputs.get("widgetJSON").getAsString();
+        var callback = jsonInputs.get("callback") == null? null: jsonInputs.get("callback").getAsString();
```

Comment:
I believe you can use jsonInputs.has("callback")
Space before ?
This occurs 5 times below